plugins/discourse-lazy-videos/lib/lazy-videos/lazy_vimeo.rb

```diff
@@ -7,7 +7,8 @@ class Onebox::Engine::VimeoOnebox
   alias_method :default_onebox_to_html, :to_html
 
   def to_html
-    if SiteSetting.lazy_videos_enabled && SiteSetting.lazy_vimeo_enabled
+    if SiteSetting.lazy_videos_enabled && SiteSetting.lazy_vimeo_enabled &&
+         !oembed_data[:uri].match(%r{videos/\d+:(.)+})
```

Comment:
I don't think the capture group is needed? And since we only need a boolean, `match?` is more appropriate.

```suggestion
         !oembed_data[:uri].match?(%r{videos/\d+:.+})
```